### ISSUE TITLE
unreachable lights not greyed-out

### ISSUE THREAD

**Issue by jandegr** [Hyrules/WinHue#268]
On older versions of Winhue the unreachable lights would be shown greyed-out. The youngest version with this behaviour I found is 3.0.4334.0 as shown below.

![winhue1](https://user-images.githubusercontent.com/9937104/70430593-6b329c80-1a7b-11ea-9762-8d817fb92579.PNG)

I installed a few versions and the oldest version I found with the new behaviour is 3.0.4666.0 as shown below.
![winhue2](https://user-images.githubusercontent.com/9937104/70430693-99b07780-1a7b-11ea-9ede-61efe3ff91bb.PNG)

All testing was done with a bridge with the following properties

modelid | "BSB002"
datastoreversion | "88"
swversion | "1935144020"
apiversion | "1.35.0"

The lights shown are philips hue and I did not change the osram bypass setting, it seems to be unchecked at install. The bridge api correctly shows reachable: false (or true where applicable).  

I also want to thank you for the great work you do in creating and maintaining WinHue

THX.

**Comment by Hyrules** [comment]
Yeah it's not a bug it's just that I removed the images for the unreachable lights and never actually implemented a proper software version. I will try to add this to the next version.
